src/main/kotlin/io/hsar/wh40k/combatsimulator/model/UnitInstance.kt

```diff
@@ -80,13 +83,15 @@ class UnitInstance(
     }
 
     private fun receiveDamage(damage: Int) {
+        println("$name took $damage damage")
         when(val health = currentAttributes.getValue(CURRENT_HEALTH)) {
             is NumericValue -> currentAttributes[CURRENT_HEALTH] = NumericValue(health.value - damage)
             else -> throw IllegalStateException("Current health ought to be a NumericValue")
         }
     }
 
-    companion object {
+    companion object : Loggable {
+        val log = logger()
```

Comment:
Unused logger?

Reply:
Will be used in the future if we can get logging config working

Reply:
Can remove for now if you like?

Reply:
Yeah, remove it for now. It's a bit misleading.